src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py

```diff
@@ -299,6 +313,9 @@
 -   name: Delete a resource group.
     text: >
         az group delete -n MyResourceGroup
+-   name: Delete a resource group.
+    text: az group delete --no-wait  --yes  --name MyResourceGroup
+    crafted: true
```

Comment:
This description is lacking. Inclusion of `--yes` means this could be considered a dangerous operation.